src/ontology/uberon-edit.obo

```diff
@@ -224035,6 +224035,16 @@ is_a: UBERON:0009569 ! subdivision of trunk
 relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date "2023-05-16T06:47:28Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480062
+name: femoral head cartilage
+def: "Hyaline articular cartilage covering the head of femur." [PMID:36292459]
```

Comment:
Consider the following definition:
"A hyaline articulate cartilage that covers the head of femur."
Consider adding a small sentence [about the function](https://europepmc.org/article/MED/32921220#:~:text=the%20femoral%20head%20cartilage%20is%20permanent%20and%20provides%20articulating%20surfaces%20of%20hip%20joint) in the definition.

Reply:
@aleixpuigb OK, thanks, I will do